batch/src/main/java/org/frankframework/batch/RecordXmlTransformer.java

```diff
@@ -84,7 +84,9 @@ public String handleRecord(PipeLineSession session, List<String> parsedRecord) t
 			}
 			Message message = new Message(xml);
 			ParameterValueList pvl = paramList==null?null:paramList.getValues(message, session);
-			return transformerPool.transform(message.asSource(), pvl);
+			try(Message transformedMessage = transformerPool.transform(message, pvl)) {
```

Comment:
Moet dit niet `message.asSource` zijn?

Reply:
Dat gebeurd intern in de transform methode?